src/lib/resolver.js

```diff
@@ -70,12 +70,14 @@ function subPlaceable(locals, ctx, lang, args, id) {
   try {
     res = resolveIdentifier(ctx, lang, args, id);
```

Comment:
Perhaps you could `const [newLocals, value] = resolveIndentifier(…);` here to avoid `res[1]` below?